less/admin/AdminNav.less

```diff
@@ -41,16 +41,10 @@
 }
 
 @media @tablet {
-  .item-search{
-    display: none;
-  }
-
-  .ExtensionItem, .item-search {
-    display: none !important;
-  }
-
-  .ExtensionListTitle {
-    display: none !important;
+  .AdminNav {
+    .item-search, .ExtensionListTitle, li[class^="item-extension"] {
```

Comment:
could we also include `.AdminLinkButton-description` ? for this case scenario:

![flarum lan_admin (8)](https://user-images.githubusercontent.com/20267363/100213287-b7861f00-2f0e-11eb-865a-846c53101a68.png)

Also, I think we should probably do the same with `li` items for category names instead of targetting `ExtensionListTitle`, just so that their margins don't add needless width to the nav element :p